oeps/processes/oep-0001.rst

```diff
@@ -328,7 +328,11 @@ be merged to capture a set of edits, and to make the proposal more visible to
 community comment. From that point, additional pull requests can be opened to
 edit the OEP, until it converges to being either "Accepted" or "Rejected".
 
-If an OEP has Draft or Under Review status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Under Review (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
+When an OEP merges with any non-final status, use the References section to
+link to a page which gives readers an updated status. For Provisional OEPs which
```

Comment:
Can we say the page is a wiki page, to give people some guidance? Or are there other possibilities?

Reply:
I would rather specify the key than the value if that makes sense, e.g. what field will have the link? Can it have a consistent name such as "Rollout Plan" or "Working Plan"? I used "References" in the language here just to follow precedent.